config/rbac/role.yaml

```diff
@@ -5,6 +5,37 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resourceNames:
+  - instascale-config
+  resources:
+  - configmaps
+  - nodes
```

Comment:
I don't think that's correct, is it? It'll only grant access the the `instascale-config` Node?

Reply:
Apologies yes, I'll remove this:
```
 resourceNames:
  - instascale-config
```

Reply:
Thanks @astefanutti 
This has been addressed in [a1a6f](https://github.com/project-codeflare/instascale/pull/139/commits/a1a6f8fa8d84b194008a9bb198a7e6c3cf810f20)